src/java.base/unix/native/libjava/FileDescriptor_md.c

```diff
@@ -144,6 +144,9 @@ static char* fmtaddr(char *buf, const char *end, unsigned char* addr, int len) {
     return buf;
 }
 
+#ifdef __APPLE__
```

Comment:
Should this apply only on Macs, or would that attribute be useful for other platforms as well?

Reply:
Agree